### DISCUSSION TITLE
Can't Install Hubs Cloud on AWS - "Rollback Failed" errors

### DISCUSSION THREAD

**Discussion by jl3471** [mozilla/hubs#2745] [General Questions & Discussion] [answered]
Hello, I followed all the steps carefully for setting up Hubs Cloud, and it seems to be successful, but then I get various errors, ultimately with a ROLLBACK_FAILED error.  See image below.
![image](https://user-images.githubusercontent.com/68816979/88971831-970cc780-d282-11ea-8690-b4e7cdd29f50.png)

Any help appreciated.

**Comment by InfiniteLee** [comment]
Try following the steps outlined here to determine what the underlying error actually is: https://hubs.mozilla.com/docs/hubs-cloud-aws-troubleshooting.html#my-aws-stack-says-rollback-complete-after-deploying-what-went-wrong

**Reply by robinkwilson** [reply]
^^ scroll down to the **first** error message.

**Comment by jl3471** [comment]
I tried once again and this time it seemed to be a successful install.  But when I try to go to the URL, it says "This site can't be reached.  Server IP address could not be found.  DNS_PROBE_FINISHED_NXDOMAIN.   If I'm using my down domain, do I need to update my nameservers?  I added the two CNAMEs and it says Success in the Validation status for the Amazon certificate page.  Do I need to do anything else for DNS settings?

**Comment by jl3471** [comment, marked as answer]
Well, I bought a 3rd amazon route 53 domain instead of using my own domain and it seems like I got it to work.  You can close this ticket.

**Reply by robinkwilson** [reply]
Happy to hear you're up and running! We're looking at improving the docs so we appreciate your help. 

Fixes:
- Disable all transfer locks on registered domains
- [Domain Recipe 3](https://hubs.mozilla.com/docs/hubs-cloud-aws-domain-recipes.html#recipe-3-domain-can-not-be-on-route-53) is only for subdomains with other programs running when root domain is occupied (aka, you can't change nameservers or your existing site will break)
- For domain registrars that you CAN change nameservers for follow the docs for **"Not set up on Route 53 and want to be?"** in [Hubs Domain Recipes](https://hubs.mozilla.com/docs/hubs-cloud-aws-domain-recipes.html)
